Assets/Scripts/UI/MainMenu/MainMenuController.cs

```diff
@@ -61,6 +61,7 @@ void Start()
 
         SelectControl(defaultButton);
         aiButtonOriginalPosition = aIButton.transform.localPosition;
+        PlayerInputManagerController.Singleton.MatchHasAI = false;
```

Comment:
Kinda disagree on not keeping this preference, but do whatever I suppose ╮(─▽─)╭